### DISCUSSION TITLE
npm ci got stucked

### DISCUSSION THREAD

**Discussion by SARVAGYSHAH23** [mozilla/hubs#4236] [General Questions & Discussion]
**Description**
npm ci got stucked at emojione@2.2.7
even after trrying multiple times.

![npmIssue](https://user-images.githubusercontent.com/68321270/117314541-7e31e200-aea4-11eb-8bae-7673a9014046.PNG)

**Comment by brianpeiris** [comment]
Converted to a discussion, since this is probably more of a troubleshooting thing.

What versions of node.js and npm are you using? You might want to try upgrading if they are very old.
You may also want to check that you aren't running out of disk space.

Lastly, it could be that `npm ci` is  just taking a long time to download things. You can check for network traffic in your task manager to confirm.

**Reply by bharathreddyza** [reply]
hey i had the same issue on my mac terminal froze at emojione ,i have latest stable version of node and npm and no problem with disc storage 
npm ci doesnt seem to work some how, might be some problem with package.lock.json
npm install is working perfectly but might have to do lot of manual version checks .

**Reply by pattersonbl2** [reply]
What type of mac do you have because i just did a fresh install in my M1 mac without any issues. I actually didn't see a section in the logs for emojione. Testing on intel now.

**Reply by bharathreddyza** [reply]
MacBook pro intel. tried multiple times freezes at the same spot
npm install is working perfectly and i am able to login and run locally  but i am not able to deploy the build with npm run deploy 
gets stuck at 
  ⠦ Building Client.

**Reply by pattersonbl2** [reply]
> MacBook pro intel. tried multiple times freezes at the same spot
> 
> npm install is working perfectly and i am able to login and run locally  but i am not able to deploy the build with npm run deploy 
> 
> gets stuck at 
> 
>   ⠦ Building Client.
> 
> 
> 
With my intel install, it took45 mins to install but never got stuck. What versions of npm are you using ?  What do you mean by you are able to log in and run locally?

**Comment by jbshin-gemiso** [comment]
I am having the same problem. Any solution?
